src/test/fuzz/parse_univalue.cpp

```diff
@@ -18,6 +18,7 @@ void initialize_parse_univalue()
 
 FUZZ_TARGET(parse_univalue, .init = initialize_parse_univalue)
 {
+    if (buffer.size() > 10'000) return;
```

Comment:
Perhaps we move this check down to right before the descriptor related code? Afaict, there's no need to restrict the testing of the json parsing to 10k bytes.

As a side note, this test does too many things at once in my opinion and should probably be split up.

Reply:
Nice